oak_functions/abi/src/lib.rs

```diff
@@ -48,6 +49,15 @@ pub mod proto {
     }
 }
 
+/// Holds the `raw_label` and the `value` to report a metric.
+#[derive(Serialize, Deserialize)]
+pub struct ReportMetricRequest {
+    pub raw_label: Vec<u8>,
```

Comment:
I think this can just be `label: String`, which would let the serialization process validate that the bytes represent a string.

Reply:
Beautiful, thank you!